django_evaluation/settings/local.py

```diff
@@ -241,17 +241,11 @@ def _set_favicon(html_color: str, project_root: Path) -> None:
 SERVER_EMAIL = os.environ.get("SERVER_EMAIL", "freva@dkrz.de")
 DEFAULT_FROM_EMAIL = SERVER_EMAIL
 
-EMAIL_HOST = os.environ.get("EMAIL_HOST", "mailhost.dkrz.de")
-
-EMAIL_PORT = int(os.environ.get("EMAIL_PORT", "25"))
-if EMAIL_PORT == 25:
-    EMAIL_HOST_USER = ""
-    EMAIL_HOST_PASSWORD = ""
-else:
-    email_secrets = _read_secret()
-    EMAIL_HOST_USER = email_secrets.get("username")
-    EMAIL_HOST_PASSWORD = email_secrets.get("password")
-
+EMAIL_HOST = "smtp.sendgrid.net"
```

Comment:
it seems all vars in [.env](https://github.com/FREVA-CLINT/freva-web/blob/sendgrid/.env) are just getting used for development side as i see in [Makefile](https://github.com/FREVA-CLINT/freva-web/blob/sendgrid/Makefile#L7), yes? what do you think to define  `EMAIL_HOST` and `EMAIL_PORT` in `.env` for dev purposes and also define these two new vars, in config playbook of `freva-deployment-web` and turn the changes back to dynamic one? the same idea for other email related vars in `settings.py`

ping @Karinon

Reply:
I just had a look at the deploy files, I think the most appropriate place for deployment is defining them in the `freva_web_conf.toml` file. As I found, we have `contacts` email and so on there . What do u think?

Reply:
Actually, I don't want to configure this at all - the less we have to fiddle with this the better.

Reply:
alright, and then it means other freva instances in other institutes has to use SendGrid and define their own SendGrid secret key and that's all. resolved

Reply:
no, not really, all instances use the same API key. We set the API here and all instances use it - it we surpass the magic threshold of 100 emails/month (which I think we won't) then we'll reevaluate.